mesonbuild/backend/ninjabackend.py

```diff
@@ -1500,13 +1495,13 @@ def generate_dynamic_link_rules(self, outfile):
                 outfile.write(description)
                 outfile.write('\n')
         outfile.write('\n')
+        args = [ninja_quote(quote_func(x)) for x in self.environment.get_build_command()] + [
+                '--internal',
```

Comment:
__[flake8]__
* [E126] continuation line over-indented for hanging indent

<!-- Comment by [SideCI](https://sideci.com) -->